.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Che Dashboard
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
```

Comment:
I think such copied/pasted description could be omitted and find in github docs on demand